### ISSUE TITLE
Append Blob changes are pushed as Put Block without append parameter

### ISSUE THREAD

**Issue by shraboin** [microsoft/AzureStorageExplorer#624]
**Storage Explorer Version**: 1.4.1
**Platform/OS Version**: Windows 10

#### Repro Steps ####
1. I gave myself a SAS that uses an Access Policy with permissions (Read/List/Add). I then created an Append blob as shown:
   ![appendblobcreate](https://user-images.githubusercontent.com/37669678/44879946-b226c500-ac70-11e8-81a9-b4bb639e4ae2.jpg)
2. When I then go to my SAS attached storage and try to append:
   ![appendingblob](https://user-images.githubusercontent.com/37669678/44879991-cd91d000-ac70-11e8-8d5d-071b23db504e.jpg)
3. It gives this prompt:
   ![prompt](https://user-images.githubusercontent.com/37669678/44880041-e8644480-ac70-11e8-81c5-3bdd6bda2005.jpg)
4. After clicking apply on “Upload” results in a 403 failure:
   ```json
   {
     "name": "StorageError",
     "message": "This request is not authorized to perform this operation using this permission.\nRequestId:212c2940-601e-0070-50a3-4039da000000\nTime:2018-08-30T20:51:36.5636032Z",
     …
   }
    ```

### Expected Result ###
From the Fiddler Trace it is doing a put blob opposed to a append blob:
```
PUT https://shraboinarmstorage.blob.core.windows.net:443/contributorscontainer2/testSharedAcess.txt?sv=2018-03-28&si=test&sr=c&sig=<SIG>&api-version=2018-03-28 HTTP/1.1
x-ms-blob-type: AppendBlob
```
If it was an append blob it would have ?comp=appendblock added to the end: 
```
https://docs.microsoft.com/en-us/rest/api/storageservices/append-block
```
Is having the Append Blob operation on the road map?
Info on append: https://docs.microsoft.com/en-us/rest/api/storageservices/constructing-a-service-sas#specifying-permissions

**Comment by craxal** [comment]
It's likely this is just a bug, as the client library we use should have support for uploading to append blobs. We'll have to investigate how easy this will be.

**Comment by MRayermannMSFT** [comment]
Verify that this scenario works well with azcopy. Close if it does.

**Comment by JasonYeMSFT** [comment]
We need to pass the proper options to azcopy to make this work properly.

**Comment by JasonYeMSFT** [comment]
An workaround has been implemented and should have been released in 1.13.0 and 1.13.1.
